profiles/config.dev-ppp

```diff
@@ -1,5 +1,5 @@
 # PIS config
-OTOPS_EFO_RELEASE_VERSION=v3.62.0
+OTOPS_EFO_RELEASE_VERSION=v3.65.0
 OTOPS_ENSEMBL_RELEASE_VERSION=111
 OTOPS_CHEMBL_RELEASE_VERSION=34
 OTOPS_PIS_RELEASE_VERSION=24.06dev
```

Comment:
I would change this as well, to 24.06

Reply:
As we discussed, we're leaving it there and removing it in a profile dedicated to 24-06 release.